### PR TITLE
Locale kit

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: scripts/locale-kit.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    await fs.writeFile(target, JSON.stringify(messages, null, 4), 'utf-8')
+}
+
+async function findAllUnusedKeys() {
+    const usedKeys: string[] = []
+    const keys = _.keys(await readMessages('en'))
+    for await (const file of walk(SOURCE_PATH)) {
+        const content = await fs.readFile(file, 'utf-8')
+        const filteredKeys = keys.filter(
+            (key) => content.includes(`t('${key}'`) || content.includes(`i18nKey="${key}"`),
```
Comment (Jack-Works) [inline scripts/locale-kit.ts:35]: Did you consider there might be line breaks?
```ts
t(
'...'
)
```

Did you consider the use of "`" symbol?

Comment (septs) [reply scripts/locale-kit.ts:35]: use tsc api?

Comment (Jack-Works) [reply scripts/locale-kit.ts:35]: use `tsc` API will be super slow.

**Review by guanbinrui — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

File: scripts/locale-kit.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import { promises as fs } from 'fs'
+import _ from 'lodash'
```
Comment (guanbinrui) [inline scripts/locale-kit.ts:2]: This pkg doesn't list in the package.json use `lodash-es` instead?

Comment (septs) [reply scripts/locale-kit.ts:2]: the is run in node.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**